src/xGadgetron/pGadgetron/Gadgetron.py

```diff
@@ -1590,3 +1601,45 @@ def calc_rpe_dcw(ad):
     dcw.fill(density_weight)
     
     return dcw
+
+def extract_calibration_data(ad):
+    '''
+    Function to extract all Acquisitions from an AcquisitionData object that
+    carry the coil calibration flag.
+    '''
+
+    if not isinstance(ad, AcquisitionData):
+        raise AssertionError("The trajectory you asked for is not among the available trajectoryies")
+
+    ref_flag = 20
+    ref_img_flag = 21
```

Comment:
can you add some of the comments if what you wrote in the PR description here?

Reply:
Done